spec/services/allocation_summary_service_spec.rb

```diff
@@ -1,10 +1,12 @@
 require 'rails_helper'
 
 describe AllocationSummaryService do
+  # TODO: - Get feedback on test case
```

Comment:
@rossjones - want to get your opinion on this - as we are not creating case information for offenders in advance they will all end up in the 'Awaiting info' bucket

Reply:
Can we create some seed data so that once deployed to staging we still have _some_ in allocated and awaiting allocation?  This will still probably break some tests but at least we can just tweak them

Reply:
Yep can definitely add seeding data to staging - 
We can seed the Test db also

Reply:
🆒